src/Core/AspectKernel.php

```diff
@@ -58,6 +59,13 @@
     protected static $containerClass = 'Go\Core\GoAspectContainer';
 
     /**
+     * Default class name for cache path resolver, can be redefined in children
+     *
+     * @var string
+     */
+    protected static $cachePathResolverClass = 'Go\Instrument\ClassLoading\CachePathResolver';
```

Comment:
It will be better to use DI here, `AspectKernel` shouldn't know anything about classes or anything else. So, please, remove all related stuff to the `GoAspectContainer` and make a definition there.